conf/returns.routes

```diff
@@ -21,4 +21,4 @@ GET   /pay-charge-in-return              uk.gov.hmrc.cgtpropertydisposalsfronten
 
 GET   /check-if-return-was-sent          uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.CheckAllAnswersAndSubmitController.submissionError()
 POST  /check-if-return-was-sent          uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.CheckAllAnswersAndSubmitController.submissionErrorSubmit()
-GET   /return-draft-confirmation         uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.ConfirmDraftReturnController.confirmDraftReturn()
+GET   /exit/return-saved                 uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.ReturnSavedController.draftReturnSaved()
```

Comment:
change endpoint to `/exit/draft-return-saved`

Reply:
the url came from Jean - I'll leave it like that unless Jean wants it to be changed